pkg/mark/renderer/blockquote.go

```diff
@@ -48,27 +49,54 @@ func (m BlockQuoteLevelMap) Level(node ast.Node) int {
 	return m[node]
 }
 
+type BlockQuoteClassifier struct {
+	patternMap map[string]*regexp.Regexp
+}
+
+func LegacyBlockQuoteClassifier() BlockQuoteClassifier {
+	return BlockQuoteClassifier{
+		patternMap: map[string]*regexp.Regexp{
+			"info": regexp.MustCompile(`(?i)info`),
+			"note": regexp.MustCompile(`(?i)note`),
+			"warn": regexp.MustCompile(`(?i)warn`),
+			"tip":  regexp.MustCompile(`(?i)tip`),
+		},
+	}
+}
+
+func GHAlertsBlockQuoteClassifier() BlockQuoteClassifier {
+	return BlockQuoteClassifier{
+		patternMap: map[string]*regexp.Regexp{
+			"info": regexp.MustCompile(`(?i)^\!info|important`),
```

Comment:
```suggestion
			"info": regexp.MustCompile(`(?i)^\!(info|important)`),
```
I believe you need to group this here, otherwise it will match only on important and not on !important

Reply:
This was tested, as can be seen by the added tests. The group is there to be able to capture it. I do not intend to do anything with groups so why add it.
![image](https://github.com/user-attachments/assets/573d8526-800e-4901-bcc9-715179b992d6)

Reply:
It is true my bad will fix